lib/ts/station/FSM.ex

```diff
@@ -2,12 +2,14 @@ defmodule Station.FSM do
   @moduledoc """
   Provides implementation of the core logic of a Station.
   """
-
+  require Logger
+  alias Util.Itinerary, as: Itinerary
   use Fsm, initial_state: :start, initial_data: []
   @behaviour Station.FSMBehaviour
 
   # Module interface definition
   def initialise_fsm(input = [_station_struct, _dependency]) do
+    Logger.info(fn -> "Initialised the fsm" end)
```

Comment:
The StationStruct has station_number and station_name. Both can be added to this log message.